components/Books.js

```diff
@@ -379,53 +403,59 @@ class Books extends React.Component {
               {this.renderContent(this.state.option)}
             </div>
             {this.state.isDuplicate ? (
-              <div
-                class="cdx-message cdx-message--block cdx-message--warning"
-                aria-live="polite"
-                style={{ marginTop: "20px", display: "inline-block" }}
-              >
-                <span class="cdx-message__icon"></span>
-                <div class="cdx-message__content">
-                  A file with this identifier{" "}
-                  <a href={`https://archive.org/details/${this.state.IATitle}`}>
-                    (https://archive.org/{this.state.IATitle})
-                  </a>{" "}
-                  already exists at Internet Archive. Please enter a different
-                  identifier to proceed.
-                  <div className="cdx-text-input input-group">
-                    <span className="input-group-addon helper" id="bid">
-                      https://archive.org/details/
-                    </span>
-                    <input
-                      className="cdx-text-input__input"
-                      type="text"
-                      id="IAIdentifier"
-                      name="IAIdentifier"
-                      onChange={(event) =>
-                        this.setState({ IAIdentifier: event.target.value })
-                      }
-                      required
-                      placeholder="Enter unique file identifier"
-                    />
-                  </div>
-                </div>
-              </div>
+              <ChangeIdentifier
+                description={
+                  <>
+                    A file with this identifier{" "}
+                    <a
+                      href={`https://archive.org/details/${this.state.IATitle}`}
+                    >
+                      (https://archive.org/{this.state.IATitle})
+                    </a>{" "}
+                    already exists at Internet Archive. Please enter a different
+                    identifier to proceed.
+                  </>
+                }
+                inputPlaceholder="Enter unique file identifier"
+                onIdentifierChange={(event) =>
+                  this.setState({ IAIdentifier: event.target.value })
+                }
+              />
             ) : null}
+
+            {this.state.isValidIdentifier === false ? (
+              <ChangeIdentifier
+                description={
```

Comment:
Please change the description to:

"The file you want to upload with title - {" "} {this.state.IATitle} either contains special characters or exceeds 50 characters in length. Please provide an identifier that consists only of letters (A-Z) and numbers (0-9)."

Reply:
@coderwassananmol 
I have corrected the description